### ISSUE TITLE
​Dot style pen pointer

### ISSUE THREAD

**Issue by AlxndrAntnv** [toeverything/AFFiNE#9158]
### Description

Now the pen pointer looks like cross, while dot style pen pointer needs less space on the screen, and is less distracting while your write or draw.

### Use case

Any time you write or draw you see a fancy dot but not a frightening cross.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by karl-tran-2002** [comment]
I agree with this issue to. Please make pointer becomes a Dot rather than a Cross, It will improve user experience.
